CryptoPrices/Sources/App/Dependencie Injection/Container+Injection.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Container+Injection.swift
+//  CryptoPrices
+//
+//  Created by Minh Pham on 21/12/2022.
+//
+
+import Factory
+import Home
+import MyCoin
+import NetworkExtension
+import UseCaseProtocol
+import NetworkCore
+import Pilot
+import Repositories
+import RepositoryProtocol
+import Styleguide
+import SwiftUI
```

Comment:
Just a minor thing, I believe we can omit these 2 imports `Styleguide` and `SwiftUI` here 🙏

Reply:
you are right, nice catch! Fixed on [5aaaed2](https://github.com/nimblehq/nimble-crypto-ios/pull/65/commits/5aaaed210a7d1f7c1fbb6c1f0ecc2ef054c5dba3)